MANIFEST/keithley2400.manifest.yaml

```diff
@@ -0,0 +1,13 @@
+
+COMMAND:
+      - {
+          name: KEITHLEY2400,
+          key: KEITHLEY2400,
+          type: KEITHLEY2400,
```

Comment:
If this belongs to `KEITHLEY2400` category then make it's key unique from file key in `COMMANDS_MANIFEST.ts` file and put it here for `type` key. For example: `KEITHLEY2400_INS`. 

Same applies for all other nodes...